src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3540,6 +3540,34 @@
     public static final int COMMENT_CONTENT =
             GeneratedJavaTokenTypes.COMMENT_CONTENT;
 
+    /**
+     * Java 14 Pattern definition.
+     *
+     * <p>For example:</p>
+     * if (obj instanceof String str) { }
+     * <pre>
+     * </pre>
```

Comment:
```suggestion
     * <pre>
     * if (obj instanceof String str) { }
     * </pre>
```

Reply:
Good catch.  This exists in #8293 as well, so I will fix it there.